src/KM_Game.pas

```diff
@@ -1513,16 +1534,20 @@ procedure TKMGame.UpdateGame(Sender: TObject);
                       //Don't autosave if the game was put on hold during this tick
                       if fGameTickCount mod 600 = 0 then
                       begin
-                        if IsMultiplayer then
-                        begin
-                          if fNetworking.IsHost then
+                        if IsMultiplayer and fNetworking.IsHost then
```

Comment:
Why did you change the logic for saving games? The change you made means that ALL clients will send the gic_GameAutoSave command (if they have autosave enabled in settings) and thus the autosave will be created 8 times on each client! :-)

Only the host is supposed to send this command since they coordinate autosaves to be synchronised.